xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -1271,11 +1271,18 @@ void CXBMCApp::SetSystemVolume(float percent)
 
 void CXBMCApp::onReceive(CJNIIntent intent)
 {
-  if (!g_application.IsInitialized())
+  std::string action = intent.getAction();
+  android_printf("CXBMCApp::onReceive - Got intent. Action: %s", action.c_str());
+
+  // Most actions can be processed only after the app is fully initialized,
+  // but some actions should be processed even during initilization phase.
+  if (!g_application.IsInitialized() && !(action == CJNIAudioManager::ACTION_HDMI_AUDIO_PLUG))
```

Comment:
```suggestion
  if (!g_application.IsInitialized() && action != CJNIAudioManager::ACTION_HDMI_AUDIO_PLUG)
```

It seems to be equivalent and a more common code

Reply:
Thank you.